### PR TITLE
Mes 4748 you have not completed the speed requirements

### PR DESCRIPTION
**Pull request by appleabs**
## Description
Mes 4748 you have not completed the speed requirements
## Checklist

- [x] PR title includes the JIRA ticket number
- [ ] Branch is rebased against the latest develop
- [x] Code has been tested manually
- [x] PR link added to JIRA ticket
- [ ] One review from each scrum team
- [ ] Squashed commit contains the JIRA ticket number

## Screenshots (optional)
Screenshot 2020-02-03 at 16 08 49

### CODE REVIEW

**Review by edwardwilson — APPROVED** [review]

File: src/pages/test-report/cat-a-mod1/components/speed-check-modal/speed-check-modal.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import { speedCheckLabels } from '../../../../../shared/constants/competencies/cata-mod1-competencies';
+
+@IonicPage()
+@Component({
+  selector: 'speed-check-modal',
+  templateUrl: 'speed-check-modal.html',
+})
+export class SpeedCheckModal {
+
+  speedCheck: speedCheckLabels[];
```
Comment (szabikr) [inline src/pages/test-report/cat-a-mod1/components/speed-check-modal/speed-check-modal.ts]: `speedChecksNeedCompleting` for example would be a better name for this property

File: src/pages/test-report/cat-a-mod1/components/speed-check-modal/speed-check-modal.html [hunk 2]
```diff
@@ -0,0 +1,23 @@
+<ion-card class="modal-alert">
+  <modal-alert-title>You've not completed the <br />speed requirements</modal-alert-title>
+  <ion-row justify-content-center>
+    <ion-col no-padding col-80>
+      <div class="modal-alert-message mes-data">
+        <div class="mes-data">To end this test, complete:</div>
+         <ul> 
+           <li *ngFor="let speed of speedCheck">{{speed}}</li>
```
Comment (szabikr) [inline src/pages/test-report/cat-a-mod1/components/speed-check-modal/speed-check-modal.html]: `speed` could be `speedCheckLabel`

**Review by szabikr — APPROVED** [review]
Just 2 simple comments if you would like to change them

File: src/pages/test-report/cat-a-mod1/components/speed-check-modal/__tests__/speed-check-modal.spec.ts [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+  beforeEach(async(() => {
+    fixture = TestBed.createComponent(SpeedCheckModal);
+    component = fixture.componentInstance;
+    component.onCancel = () => {
+    };
+    component.onTerminate = () => {
+    };
+  }));
+
+  describe('Class', () => {
```
Comment (TomBillingtonUK) [inline src/pages/test-report/cat-a-mod1/components/speed-check-modal/__tests__/speed-check-modal.spec.ts]: you can remove this describe

**Review by TomBillingtonUK — APPROVED** [review]
Looks good

**Pull request merged**